components/shared/workCard/WorkCard.tsx

```diff
@@ -4,39 +4,37 @@ import React from "react"
 
 import { WorkTeaserFragment } from "@/lib/graphql/generated/fbi/graphql"
 
-import { displayCreators } from "./helper"
 import WorkCardAvailabilityRow from "./WorkCardAvailabilityRow"
+import { displayCreators } from "./helper"
 
 type WorkCardProps = {
   work: WorkTeaserFragment
 }
 
 const WorkCard = ({ work }: WorkCardProps) => {
   return (
-    <div className="mb-4">
-      <Link href={`/work/${work.workId}`}>
-        <div key={work.workId} className="rounded-sm bg-background-overlay p-2 md:p-4">
-          <div
-            className="px-auto shadow-coverPicture relative mx-auto mb-3 mt-6 flex aspect-[166/228] w-[calc(100%-76px-4px)]
-              items-center rounded-sm md:mb-6 md:mt-9 md:w-[calc(100%-106px)]">
-            <Image
-              // TODO: dynamic image
-              src="https://i.pinimg.com/564x/b0/4a/b2/b04ab2de27591aee4a76015e42e0e282.jpg"
-              alt="work image"
-              layout="responsive"
-              width={166}
-              height={228}
-              className="overflow-hidden rounded-sm"
-            />
-          </div>
-          <WorkCardAvailabilityRow materialTypes={work.materialTypes} />
+    <Link href={`/work/${work.workId}`}>
```

Comment:
I think we should have a global route helper function. We probably will need the work route in several places.
So it could eg. be called `resolveWorkUrl(work: Work)`

Reply:
I will create a ticket on this